pytorch_lightning/core/lightning.py

```diff
@@ -1085,19 +1086,84 @@ def manual_backward(self, loss: Tensor, optimizer: Optimizer, *args, **kwargs) -
 
         .. tip:: In manual mode we still automatically clip grads if Trainer(gradient_clip_val=x) is set
 
+        .. tip:: In manual mode we still automatically accumulate grad over batches if Trainer(accumulate_grad_batches=x) is set
```

Comment:
check the rendering but I think it shall starts on a new line

Reply:
<img width="883" alt="Screenshot 2020-11-10 at 08 34 11" src="https://user-images.githubusercontent.com/12861981/98650101-5831e780-2330-11eb-9902-f9b46f590973.png">